clothing_app/clothing_app/spiders/ackermans.py

```diff
@@ -0,0 +1,241 @@
+import copy
+import json
+from urllib.parse import urljoin
+
+import scrapy
+
+from ..items import ProductItem, SizeItem
+
+
+class AckermansSpider(scrapy.Spider):
+    name = "ackermans"
+
+    def start_requests(self):
+        yield scrapy.Request(self.navigation_api, self.parse_home_page)
+
+    navigation_api = "https://www.ackermans.co.za/api/startup-data.json"
```

Comment:
These class variables should be placed before other functions so that we don't mix them with the function body.